rbc/omniscidb.py

```diff
@@ -703,6 +715,7 @@ def _make_udtf(self, caller, orig_sig, sig):
 
         consumed_index = 0
         name = caller.func.__name__
+        type_to_extarg = self.type_to_extarg
```

Comment:
Any particular reason for assign this rather than just use `self.type_to_extarg`?

Reply:
No. Now using `self.type_to_extarg` instead.